web/src/components/navigation/navbarmobile.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import Logo from '../../img/tinysocial-logo.png';
+import MenuIcon from '@material-ui/icons/Menu';
+import {
+  BrowserRouter,
+  Route,
+} from 'react-router-dom';
+import {
+  AppBar,
+  Toolbar,
+  Grid,
+  IconButton,
+  Drawer,
+  List,
+  makeStyles,
+  Divider,
+  Button,
+  Typography, Tab, Tabs,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  list: {
+    width: 250,
```

Comment:
Is it possible to combine these constants in another file (e.g. constants.js or something like this)? I can see same variables being used in multiple files.

Reply:
I made a constants.js file and separated an useStyles.
I can't find files that used same variables. Could you tell me which file is?